airflow/macros/__init__.py

```diff
@@ -85,3 +86,12 @@ def datetime_diff_for_humans(dt: Any, since: DateTime | None = None) -> str:
     import pendulum
 
     return pendulum.instance(dt).diff_for_humans(since)
+
+
+def deserialize(data: str) -> dict[Any, Any]:
```

Comment:
`deserialize` it is a bit broad name which could mean everything and might confuse users.